pymatgen/command_line/bader_caller.py

```diff
@@ -42,11 +44,8 @@
 __date__ = "4/5/13"
 
 
-BADER_EXE = which("bader") or which("bader.exe")
```

Comment:
`monty.dev.requires` is cleaner but would break all `mypy` tests, as decorator seems too implicit to be understood properly by `mypy` at this moment